magellan/blocker/attr_equiv_blocker.py

```diff
@@ -12,16 +14,77 @@
 
 
 class AttrEquivalenceBlocker(Blocker):
+    """Blocks two tables, a candset, or a pair of tuples based on attribute equivalence.
+    """
+
     def block_tables(self, ltable, rtable, l_block_attr, r_block_attr,
                      l_output_attrs=None, r_output_attrs=None,
                      l_output_prefix='ltable_', r_output_prefix='rtable_',
-                     verbose=True):
+                     verbose=False, show_progress=True, n_jobs=1):
+        """Blocks two tables based on attribute equivalence.
+
+        Finds tuple pairs from left and right tables such that the value of
+        attribute l_block_attr of a tuple from the left table exactly matches
+        the value of attribute r_block_attr of a tuple from the right table.
+        This is similar to equi-join of two tables.
+
+        Args:
+            ltable (pandas dataframe): left input table.
+
+            rtable (pandas dataframe): right input table.
+
+            l_block_attr (string): blocking attribute in left table.
+
+            r_block_attr (string): blocking attribute in right table. 
+
+            l_output_attrs (list of strings): list of attributes from the left
```

Comment:
list of strings can be changed to list, and mention the attribute names are strings in the explanation